realm/realm-library/src/main/java/io/realm/internal/Property.java

```diff
@@ -29,23 +29,92 @@
     public static final boolean REQUIRED = true;
     public static final boolean INDEXED = true;
 
+    @SuppressWarnings("WeakerAccess")
```

Comment:
The following fields are used internally AFAICT, they don't need to be public to be generated in the header file